### DISCUSSION TITLE
@AshtonDavies Does Skiovox work for version 121?

### DISCUSSION THREAD

**Discussion by v4pl** [3kh0/ext-remover#946] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
If so give me link to tutorial I thought it only worked on 120 and also is downgrading still possible?

**Comment by AshtonDavies** [comment]
SKIOVOX for v120 may or may not work in v121, and there is a good chance that users in Titanium Network have found other methods for higher versions.

Downgrading is patched in v116.

**Comment by v4pl** [comment, marked as answer]
Titanium Network what that? They have a discord or something

**Reply by AshtonDavies** [reply]
> Titanium Network what that? They have a discord or something

Yes.

**Reply by v4pl** [reply]
I joined the discord but I can't find anything about Skiovox

**Reply by v4pl** [reply]
@AshtonDavies plz answer

**Reply by AshtonDavies** [reply]
> I joined the discord but I can't find anything about Skiovox

Go to kajigs. That's where the exploits are at.

**Reply by v4pl** [reply]
They said nothing about 121 and they told me it was patched

**Comment by SpaceMonke135** [comment]
yeah...the skiovox might not be possible for 121
but It might work on 122

**Reply by v4pl** [reply]
Wait how does that work not on 121 but 122?

**Reply by v4pl** [reply]
122 isn't out yet correct?

**Reply by AshtonDavies** [reply]
> Wait how does that work not on 121 but 122?

SKIOVOX seems to coincidentally not work on some versions based on my memory of what I've read. It's not that it has an actual patch in efforts to prevent it that we're sure about. It just doesn't work in some versions apparently.

**Reply by v4pl** [reply]
But from what I tested they patched "esc" and "alt shift s" on version 121

**Reply by AshtonDavies** [reply]
> But from what I tested they patched "esc" and "alt shift s" on version 121

Yeah, good point. That sounds like an intentional fix.

**Reply by v4pl** [reply]
Ya so is ltmeat flood the only usable exploit for 121?

**Reply by AshtonDavies** [reply]
> Ya so is ltmeat flood the only usable exploit for 121?

The only legitimate one I'd say. There are other options such as Dextensify, which is rather trashy, or Corkey.

**Reply by v4pl** [reply]
lol they take so long to do and trash

**Reply by AshtonDavies** [reply]
And I'm not even entirely sure if the loading bug utilized in LTMEAT Flood is still unfixed or not. I don't remember the last version I had a permanent freeze test on.

**Comment by funkymokey** [comment]
ahston owns this repo now
